packages/client/hmi-client/src/components/model/tera-initial-metadata-entry.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<section>
+		<h6>{{ target }}</h6>
+		<tera-input
+			label="Name"
+			:model-value="getInitialName(model, target)"
+			@update:model-value="$emit('update-initial-metadata', { key: 'name', value: $event })"
+		/>
+		<tera-input
+			label="Description"
+			:model-value="getInitialDescription(model, target)"
+			@update:model-value="
+				$emit('update-initial-metadata', {
+					key: 'description',
+					value: $event
+				})
+			"
+		/>
+		<tera-input
+			label="Unit"
+			:model-value="getInitialUnits(model, target)"
+			@update:model-value="$emit('update-initial-metadata', { key: 'units', value: $event })"
+		/>
+		<tera-input
+			label="Concept"
+			:model-value="getInitialConcept(model, target)"
+			disabled
+			@update:model-value="
```

Comment:
Is this `disabled` actually disabling the `tera-input` component? 
It reads weird when it is disabled and has a @update:model-value enabled??

Reply:
Yeah it disables it since the attributes are binded to the input tag in tera-input. I am disabling it for now since there doesn't seem to be a way to editing/adding concepts. That `@update` is just there kinda as a placeholder I guess, just so if we figure out a way to update it, things are already hooked up